test/javascripts/acceptance/post-event-builder-test.js

```diff
@@ -0,0 +1,69 @@
+import { acceptance, query } from "discourse/tests/helpers/qunit-helpers";
+import { test } from "qunit";
+import { click, visit } from "@ember/test-helpers";
+import selectKit from "discourse/tests/helpers/select-kit-helper";
+
+acceptance("Post event - composer", function (needs) {
+  needs.user({ admin: true, can_create_discourse_post_event: true });
+  needs.settings({
+    discourse_local_dates_enabled: true,
+    discourse_calendar_enabled: true,
+    discourse_post_event_enabled: true,
+    discourse_post_event_allowed_on_groups: "",
+    discourse_post_event_allowed_custom_fields: "",
+  });
+
+  test("composer event builder", async function (assert) {
+    await visit("/");
+    await click("#create-topic");
+    await click(".toolbar-popup-menu-options .dropdown-select-box-header");
+    await click(".toolbar-popup-menu-options *[data-value='insertEvent']");
+
+    const m = ".discourse-post-event-builder-modal";
```

Comment:
Minor but same comments as https://github.com/discourse/discourse-calendar/pull/299/files#r928366796. There is very little upside in abbreviating variable names compared to its impact on readability.